Add Snapshotter CLI Documentation

SwaroopH opened the pull request:
Fixes #68 

Deploy on: https://powerloom-docs-245iq.ondigitalocean.app/build-with-powerloom/snapshotter-node/lite-node-v2/cli

## Changes Made

### New Documentation
- Added comprehensive CLI documentation at `/docs/build-with-powerloom/snapshotter-node/lite-node-v2/cli.md`
  - Installation instructions (binaries and PyPI)
  - Interactive shell mode usage
  - Complete command reference
  - Configuration management
  - Multi-node setup guide
  - Troubleshooting section
  - Migration guide from manual setup

### Updated Existing Pages
- **Lite Node V2 Getting Started**: Added "Alternative Setup Methods" section highlighting the CLI as recommended solution for multiple slots
- **Devnet Getting Started**: Added "Alternative Setup: Snapshotter CLI" section with benefits and use cases

### Bug Fixes
- Fixed broken anchor links in `build-sh-advanced.md` and `getting-started.md`
- Updated internal link reference from `#setting-up-multiple-nodes-on-a-linux-vps` to `#alternative-setup-methods`

## Impact
This documentation helps users discover and adopt the Powerloom Snapshotter CLI, which significantly simplifies:
- Multi-slot management
- Configuration setup
- Node deployment and monitoring
- Troubleshooting

The CLI is positioned as the recommended approach for operators managing multiple slots, while maintaining the existing manual setup documentation for those who prefer it.

SwaroopH committed:
fixed broken anchor links

SwaroopH committed:
Add Snapshotter CLI documentation and alternative setup methods

Introduced a new section for the Powerloom Snapshotter CLI, detailing its features, installation, and usage for managing multiple snapshotter instances. Updated the 'Getting Started' guide to reference alternative setup methods, emphasizing the benefits of using the CLI for operators with multiple slots.

anomit commented on docs/build-with-powerloom/snapshotter-node/lite-node-v2/cli.md:
```diff
@@ -0,0 +1,406 @@
+powerloom-snapshotter> list
+
+# Monitor logs in real-time
+powerloom-snapshotter> logs
```
we dont have a `logs` command for the CLI

anomit replied:
Addressed in my recent commits

anomit committed:
chore: cleanup CLI commands not implemented or refactored under different names

anomit committed:
docs: improve CLI documentation clarity and credentials management

- Standardize command section headers with proper capitalization
- Expand credential management documentation with more detailed commands
- Add 'identity list' command examples
- Clarify identity management description and capabilities
- Rename "Configuration Management" to "Credentials Management: Internals"
- Make command descriptions more descriptive and action-oriented

anomit committed:
fix: use explicit heading ID against Instsllation section for correct linkage

anomit committed:
fix: correct GitHub repository link capitalization in CLI documentation

anomit committed:
fix: standardize GitHub repository link capitalization across documentation

xadahiya reviewed (approved):
Looks good after @anomit's changes!

anomit reviewed (approved):
Good to go.

Seth-Schmidt reviewed (approved):
Looks good!
